### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/webbpsf/commit/2187c88c54afd1c2fb5c3c71fae0f4271b129954#commitcomment-62588526

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/webbpsf/commit/e5b91f62b4d598a12461ec60f70afc451dd4b2f0

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

I intend to annotate this PR shortly. I'm happy to drop changes to files, drop specific terms, or split things as requested, although this may take some time.

### CODE REVIEW

File: dev_utils/field_dependence/read_codev_dat_fits.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 from datetime import datetime
 
 # Script that reads in a set of CodeV Pupil map OPD files across fields and instruments and fits Zernikes to the OPD
-# distribution at each field point and then Lengdres to the variation of each Zernike coefficient across field.  The
+# distribution at each field point and then Legendres to the variation of each Zernike coefficient across field.  The
```
Comment (jsoref) [inline dev_utils/field_dependence/read_codev_dat_fits.py:10]: Not sure about this

File: docs/_themes/theme_webbpsf/static/bootstrap-webbpsf.css [hunk 2]
```diff
@@ -488,7 +488,7 @@ a.headerlink {
 a.headerlink:hover {
     background-color: #0069d6;
     color: white;
-    text-docoration: none;
+    text-decoration: none;
```
Comment (jsoref) [inline docs/_themes/theme_webbpsf/static/bootstrap-webbpsf.css:491]: This is likely a behavior change

File: docs/jwst.rst [hunk 3]
```diff
@@ -190,7 +190,7 @@ behavior on filter selection can be disabled by setting ``nircam.auto_channel =
     SW, otherwise for NRCA5 or NRCB5 the pixel scale will be for LW.
     If you set the ``filter`` attribute to a filter in the short wave channel,
     the pixel scale will be set for SW, otherwise for a filter in the long wave
-    challen the scale will be set for LW.
+    channel the scale will be set for LW.
```
Comment (jsoref) [inline docs/jwst.rst:193]: ?

File: docs/relnotes.rst [hunk 4]
```diff
@@ -101,7 +101,7 @@ This release only improves a subset of WFIRST functionality; additional improvem
 
     -  Imaging
     -  Grism
-    -  Prisim
+    -  Prism
```
Comment (jsoref) [inline docs/relnotes.rst:104]: ?

File: webbpsf/opds.py [hunk 5]
```diff
@@ -2864,7 +2864,7 @@ def test2_OPDbender(filename='OPD_RevV_nircam_132.fits'):
 
 class OteThermalModel(object):
     """
-    Create an object for a delta_time that predictes the WSS Hexike coeffcients
+    Create an object for a delta_time that predicts the WSS Hexike coefficients
```
Comment (jsoref) [inline webbpsf/opds.py:2867]: The alternative is `predicted`...

File: webbpsf/webbpsf_core.py [hunk 6]
```diff
@@ -1232,7 +1232,7 @@ def _apply_jitter(self,  result, local_options=None):
 
         This adds options to model JWST coarse point ("PCS=Coarse") under
         two sets of assumptions:
-            "PCS=Coarse": 67 mas Gaussian jitter, as advised by Nelan & Maghamni based on
+            "PCS=Coarse": 67 mas Gaussian jitter, as advised by Nelan & Maghami based on
```
Comment (jsoref) [inline webbpsf/webbpsf_core.py:1235]: This seems to match another instance of what I think is the same name in this repo.

**Review by shanosborne — APPROVED** [review]
Hi @jsoref , this all looks great - thank you for putting in these changes (and sorry they took so long to review)! I’ve gone through all the updates and they all look good to me. All of these changes are in comments/docstrings and won’t affect the code except for 2 changes that are in the CSS stylesheets, but I’ve confirmed they don’t really change anything when the docs are built. 

So this PR is good to go. I’m going to merge it without the CI having run because it saves the time/effort of rebasing the branch and because the CI won’t test anything anyway (since again, all the changes are in docstrings, comments, docs files, etc).

**Comment by jsoref** [comment]
Thanks @shanosborne. As you can see, I'm not in any hurry. My goals are generally to drive the world forward, both the projects to which I'm contributing and the tooling I'm using to make the contributions.

Fwiw, I'm still iterating on my tooling. Overnight, I've added enough bits to make it possible to submit the workflow to starter workflows.

I might see about doing another round of PRs to other spacetelescope repos (I have 3 left open and 5 merged), but I have a large unsorted queue of projects, and I don't have any particular prioritization to them. If I knew that certain repositories would be more likely to benefit/respond sooner, I'd gladly prioritize them.

**Pull request merged**